screens/auth/LogInScreen.js

```diff
@@ -160,55 +160,66 @@ export default class LogInScreen extends React.Component {
 
     return (
       <AuthScreenContainer>
-        <BackButton onPress={() => this.props.navigation.goBack(null)}>
-          <FontAwesome5 name="arrow-left" solid size={24} />
-        </BackButton>
-        <BigTitle>Log In</BigTitle>
-        <FormContainer>
-          <AuthTextField
-            fieldType="Phone Number"
-            value={this.state.phoneNumber}
-            changeTextCallback={async (text) => {
-              this.setState({ phoneNumber: text, error: '' });
-            }}
-            // Display error indicator ('no text') only when login fails
-            error={this.state.error ? ' ' : ''}
-          />
-          <AuthTextField
-            fieldType="Password"
-            value={this.state.password}
-            changeTextCallback={async (text) => {
-              this.setState({ password: text, error: '' });
-            }}
-            // Display error indicator ('no text') only when login fails
-            error={this.state.error ? ' ' : ''}
-          />
-          <Caption
-            style={{ alignSelf: 'center', fontSize: 14 }}
-            color={Colors.error}>
-            {this.state.error}
-          </Caption>
-        </FormContainer>
-        <JustifyCenterContainer>
-          <FilledButtonContainer
-            style={{ marginTop: 104 }}
-            color={
-              !logInPermission ? Colors.lightestGreen : Colors.primaryGreen
-            }
-            width="100%"
-            onPress={() => this.handleSubmit()}
-            disabled={!logInPermission}>
-            <ButtonLabel color={Colors.lightest}>Log in</ButtonLabel>
-          </FilledButtonContainer>
-
-          {/* TODO @tommypoa: Forgot password functionality
+        <ScrollView
+          ref={(ref) => {
+            this.scrollView = ref;
+          }}
+          style={{ flex: 1 }}
+          keyboardShouldPersistTaps="always"
+          showsVerticalScrollIndicator={false}>
```

Comment:
idk what this is but i assume it's doing the magic thing, you're an absolute legend